images/clang_tidy/Dockerfile.erb

```
@@ -0,0 +1,16 @@
+FROM sider/devon_rex_base:2.22.0
+
+COPY --chown=<%= chown %> images/clang_tidy/packages.list ${RUNNER_USER_HOME}/
+
+USER root
+RUN curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
+    add-apt-repository "deb http://apt.llvm.org/buster/ llvm-toolchain-buster-10 main" && \
+    apt-get update && \
+    xargs apt-get install -qq -y --no-install-recommends < ${RUNNER_USER_HOME}/packages.list && \
+    apt-get clean && \
+    update-alternatives --install /usr/local/bin/clang-tidy clang-tidy /usr/lib/llvm-10/bin/clang-tidy 20 && \
+    echo "analyzer_runner ALL=NOPASSWD: /usr/bin/apt-get" >> /etc/sudoers
```

[suggestion] How about the following suggestions?

- Replace `analyzer_runner` with `${RUNNER_USER}`.
- Create a new file into `/etc/sudoers.d/` instead of modifying the `/etc/sudoers` file.

```suggestion
    echo "${RUNNER_USER} ALL=NOPASSWD: /usr/bin/apt-get" > /etc/sudoers.d/runner
```

---

Good catch!
Fixed. commit 5ad9e819efea4b4dd7f5e1204d8709a87aabf4fd